components/schemes/Container/src/MDCContainerScheme.h

```diff
@@ -26,12 +26,12 @@
 /**
  The color scheme to use for the container scheme.
  */
-@property(nonatomic, nullable, readonly) id<MDCColorScheming> colorScheme;
+@property(nonatomic, nonnull, readonly) id<MDCColorScheming> colorScheme;
```

Comment:
Should we provide an initializer now that these are not null? 
How are we expecting client apps to assign these values on a newly initialized container scheme?

Reply:
@galiak11  There is an `init` method for `MDCContainerScheme` in the `.m` file. I don't think there's much value dictating initialization/construction in the protocol - implementations may use builders, factories, `init` methods, etc.   The protocol is focused on consumption - requiring the properties, their types and attributes.  

Clients can assign the properties directly to the implementation properties.

Reply:
I think we just intend to have the `init` in the `.m` file as @romoore has outlined. Do you agree with that?